src/views/ItemList/ItemList.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import { fetchItems } from '../../services/fetchdata/fetchdata';
+import Item from '../../components/Item/Item';
+import style from './ItemList.css';
+import { fetchItemByName } from '../../services/fetchdata/fetchdata';
+import { Route, useRouteMatch, Link } from 'react-router-dom';
+import ItemDetails from '../../components/ItemDetails/ItemDetails';
+
+export default function ItemList() {
+  const [fetchUrl, setFetchUrl] = useState('https://pokeapi.co/api/v2/item/');
+  const [itemList, setItemList] = useState([]);
+  const [selectedItem, setSelectedItem] = useState('');
+  const [detailItem, setDetailItem] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  const { url } = useRouteMatch();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const results = await fetchItems(fetchUrl);
+      setItemList(results);
+    };
+    fetchData();
+  }, [fetchUrl]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const results = await fetchItemByName(selectedItem);
+        setDetailItem(results);
+        setLoading(false);
+      } catch {
+        setLoading(false);
+      }
+    };
+    fetchData();
+  }, [selectedItem]);
+
+  //handle click to make go back and forth from the 20 item page.
+  const handleClick = (direction) => {
+    if (direction === 'next') {
+      setFetchUrl(itemList.next);
```

Comment:
I might check for null values here as the console throws an error when no prev / no next exist

Reply:
I ended up just hiding the button when there is a null value